realm/src/main/java/io/realm/internal/RealmCore.java

```diff
@@ -117,7 +117,7 @@ public void run(){
         });
     }
 
-    private static void initTightDB() {
+    private static void initRealm() {
```

Comment:
Since the class is `RealmCore`, the method should be named `init`: the class will be `RealmCore.init()` which is not confusing.